src/components/menuNavbar/MenuNavbar.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import { ItAcademyLogo } from '../../assets/svg';
+
+const MenuNavbar = () => {
+  const [activeItem, setActiveItem] = useState('Perfiles');
+  const menuItems = [
+    'Perfiles',
+    'Matching',
+    'Proyectos',
+    'Estadísticas',
+    'Guías útiles',
+  ];
+
+  return (
+    <nav className="hidden gap-28 w-44 flex-none px-6 py-7 md:flex md:flex-col">
+      <div>
+        <img src={ItAcademyLogo} alt="itAcademy Logo" />
+      </div>
+      <div className="flex flex-col gap-9">
+        {menuItems.map((item, index) => (
+          <button
```

Comment:
Reuse this with the standard UI library from IT Academy here: https://ui.eurecatacademy.org/?path=/docs/atoms-button--docs